tests/test_delivery.py

```diff
@@ -0,0 +1,10 @@
+from idunn.blocks.delivery import DeliveryBlock
+from idunn.places import POI
+
+
+def test_website_block():
+    web_block = DeliveryBlock.from_es(
+        POI({"properties": {"delivery": "yes", "takeaway": "yes"}}), lang="en"
+    )
+
+    assert web_block == DeliveryBlock(available=["delivery", "takeaway"])
```

Comment:
`test_website_block` and `web_block` should be renamed

Reply:
As `web_block` is used in several other blocks I was wondering if it was meaning something ^^' `test_website_block` is a plain mistake though.

Reply:
I really think this is some kind of copy-paste contamination :smile: